tests/integration/interaction/test_range_min_max.py

```diff
@@ -75,75 +56,59 @@ def pan_plot(selenium, pan_x=None, pan_y=None):
     actions.perform()
 
 
-def test_range_with_callback_triggers_alert(output_file_url, selenium):
-    # Simple test to ensure range callbacks are working
-    # Rest of tests in this file depend on range callback.
-
-    plot = make_pan_plot_with_callback()
-    initial_start = plot.x_range.start
-    save(plot)
-    selenium.get(output_file_url)
-    assert has_no_console_errors(selenium)
-
-    # Pan plot and test for new range value
-    pan_plot(selenium, pan_x=100, pan_y=100)
-    new_range_start = float(selenium.execute_script("""alert(window.get_x_range_start())"""))
-    selenium.switch_to_alert().dismiss()
-    assert new_range_start < initial_start
-
-
 def test_x_range_does_not_pan_left_of_x_min(output_file_url, selenium):
     x_range_min = -1
-    plot = make_pan_plot_with_callback(xr=Range1d(0, 3, bounds=(x_range_min, None)))
+    plot = make_plot(xr=Range1d(0, 3, bounds=(x_range_min, None)))
     save(plot)
     selenium.get(output_file_url)
     assert has_no_console_errors(selenium)
 
     # Pan plot and test for new range value
-    pan_plot(selenium, pan_x=200, pan_y=0)
-    new_range_start = float(selenium.execute_script("""alert(window.get_x_range_start())"""))
+    pan_plot(selenium, pan_x=150, pan_y=0)
```

Comment:
The tests fail without the change. One issue was that the pan end was outside of the plot frame, so that the pan event didn't happen. 

I'm not 100% sure why this wasn't a problem earlier.